pkpdapp/pkpdapp/models/combined_model.py

```diff
@@ -331,7 +331,8 @@ def create_myokit_model(self):
                 var = myokit_compartment.add_variable(
                     new_names[0],
                     rhs=myokit.Name(myokit_var),
-                    initial_value=0
+                    initial_value=0,
+                    unit=myokit_var.unit() * myokit.units.hour,
```

Comment:
Would it be safer to use `self.get_time_unit()` here, rather than assuming the model uses hours?